Nelson_CSG/CSG_XML/1888-08-08-ChTimes.xml

```diff
@@ -117,6 +117,8 @@
                <p><said who="#employer" ana="male">We spent $5 in providing seats for the clerks. They are the kind that fold under the counter, and clerks are permitted to sit down when not busy. Formerly there was a dining room in the basement, but <persName ref="#promMen">Dr. De Wolf</persName> instructed us to provide a better one for the clerks. The diningroom is on the fifth floor and is provided with long tables and benches. It will seat about fifty persons, and is lighted by one window. Dinner begins at 11 and continues until 2. Forty-five minutes is given each <choice><sic>employe</sic><reg>employee</reg></choice> for this meal. About ten are permitted to go from each floor at a time. Clerks are permitted to use the elevators to the extent of two at a time. All of the male and many of the <damage unit="word" quantity="1" agent="blurred"/><unclear><supplied resp="#nll">female</supplied></unclear> clerks fo out for dinner, so that the number who dine in the store is <damage unit="word" quantity="1" agent="blurred"/><unclear><supplied resp="#nll">comparal</supplied></unclear> very small.<!-- 2016-07-21: nll: There are no closing quotes here. --></said></p>
+               <p><said who="#employer" ana="male">Wrok begins at 7:15 and continues until 6:30. The cash girls are permitted to go at 5:30. We can not close at noon Saturday on account of the class of trade. Our customers are generally poor people who do their shopping Saturday afternoon.</said><!-- 2016-08-09: nll: There are closing quotes here. --></p>
```

Comment:
@nlottig94 I was just real quick reviewing the changed file and looks like there might be a spelling error in this line... Wrok --> Work ?? 

Thanks again for all your hard work.